wsdl/ver10/recording.wsdl

```diff
@@ -683,6 +690,32 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:attribute>
 				<xs:anyAttribute processContents="lax"/>
 			</xs:complexType>
+			<xs:element name="OverrideSegmentDurationRequest">
+				<xs:complexType>
```

Comment:
Syntax error: the request name must not include suffix "Request". Needs to change also below in the message definition.